### ISSUE TITLE
trigger effects fails with complex models

### ISSUE THREAD

**Issue by fnumatic** [domino-clj/domino#14]
Trigger **effects** works for simple flat models like:
```
{ :model  [[:n {:id :n}]
           [:m {:id :m}]]}
```

But does not work for nested models like this:
```
{:model   [:foo
             [:o {:id :o}]
             [:p {:id :p}]]}
```

Btw. trigger **events** should have the same problem.

PR follows

**Comment by nikolap** [comment]
Merged into dev, thanks for the PR

**Comment by yogthos** [comment]
just released `0.3.0` with the fix
